src/analytics.ts

```diff
@@ -174,6 +174,32 @@ export const logEvent = ({
     });
 };
 
+export const logEcommerceEvent = (
```

Comment:
I'd rename this as "logNamedEvent" or something similar.

Reply:
I'd use this semantic name for the moment. Because this will only be used for ecommerce events. In next Qs when we integrate GA4 we can review all these methods